### DISCUSSION TITLE
Unable to add family and indvidual metadata or unable to add dataset to local instance

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#3851] [Q&A]
Hi Team,

I'm trying to analyse variants present in a mother daughter sample in seqr and had uploaded the vcf in our local instance. The upload using load_data.sh was successful as you can see in the screenshot below:
![image](https://github.com/broadinstitute/seqr/assets/122073339/1ffeaa48-9e03-444b-88fb-e6d54c09a457)

After the successful upload of the VCF, I'm unable to add the metadata of the individuals or families as it shows an error "Unable to find individuals to update for any of the 2 parsed individuals. No matching ids found for 2 individuals." and at the same time I'm unable to add the index through edit datasets as well.
![image](https://github.com/broadinstitute/seqr/assets/122073339/503504b5-3e98-4072-a8c0-920b1f14520a)

![image](https://github.com/broadinstitute/seqr/assets/122073339/df9bc0f3-407a-456e-bfe2-556de7db98f8)

The metadata file, that I had created looks like this:
![image](https://github.com/broadinstitute/seqr/assets/122073339/3a90166c-9a69-46c8-ad9c-20191af921d1)

Could you please help me here and let me know what I'm missing or needs to be done?

Thanks and Regards,
Tulasi S

**Comment by hanars** [comment]
The metadata file can only update metadata for individuals that have already been created. You need to first actually create the Individuals in your project using the "Bulk Upload Individuals" functionality, not the "Bulk Upload Individual Metadata". Only after you see the individuals in your project can you update their metadata or load data for them

**Reply by Tulasi08** [reply]
Thank you so much for the clarification @hanars ! I'm able to analyse the variants successfully.
